lib/rucio/transfertool/fts3.py

```diff
@@ -460,12 +460,18 @@ def _is_recoverable_fts_overwrite_error(cls, request, reason, file_metadata):
             return False
         dst_file = file_metadata.get('dst_file', {})
         dst_type = file_metadata.get('dst_type', None)
+        record_counter('transfertool.fts3.overwrite.check.{rsetype}.{rse}.{reason}',
+                       labels={'rse': file_metadata["dst_rse"], 'rsetype': dst_type, 'reason': reason})
+
         if 'Destination file exists and overwrite is not enabled' in (reason or ''):
             if cls._dst_file_set_and_file_correct(request, dst_file):
-                if dst_file.get('file_on_tape'):
-                    return True
-                elif dst_type == 'DISK':
+                if dst_type == 'DISK' or dst_file.get('file_on_tape'):
+                    record_counter('transfertool.fts3.overwrite.ok.{rsetype}.{rse}.{reason}',
+                                   labels={'rse': file_metadata["dst_rse"], 'rsetype': dst_type, 'reason': reason})
                     return True
+
+        record_counter('transfertool.fts3.overwrite.fail.{rsetype}.{rse}.{reason}',
+                       labels={'rse': file_metadata["dst_rse"], 'rsetype': dst_type, 'reason': reason})
```

Comment:
having RSE and reason in labels seems like a potential for an explosion of number of metrics. Neither prometheus nor statstd will like it. Are you sure about recording the "reason" ?  I don't see it's usefulness. For "ok" it will always be 'Destination file exists and overwrite is not enabled'. 
Maybe move this "fail" metric inside  `if 'Destination file exists and overwrite is not enabled'` ... this way you'll always record metrics for this particular reason. 

Other than that, the PR looks good